src/main/java/de/chojo/repbot/service/RoleUpdater.java

```diff
@@ -34,7 +34,7 @@ public RoleUpdater(Guilds guilds, RoleAssigner roleAssigner) {
     public void onMessageReceived(@NotNull MessageReceivedEvent event) {
         if (!event.isFromGuild()) return;
         if (!guilds.guild(event.getGuild()).settings().general().reputationMode().isAutoRefresh()) return;
-        if (isChecked(event.getMember())) return;
+        if (event.getMember() == null || isChecked(event.getMember())) return;
```

Comment:
Based on the docs I would guess this is only the case for webhook messages. But they don't need to be processed by us anyways